deliveries/myriad_social-milestone_2.md

```diff
@@ -0,0 +1,52 @@
+
+# Milestone Delivery :mailbox:
+
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/myriad_social.md
+
+
+
+* **Milestone Number: 2** 
```

Comment:
```suggestion
* **Milestone Number:** 2 
```

Reply:
@gilangbh or @agustinustheo, could you commit this change request? It seems that I can't.